src/components/AgmLocationChange/AgmYear.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-text-field
+    id="year-txt"
+    ref="textAreaRef"
+    hide-spin-buttons
+    type="number"
+    filled
+    :label="label"
+    :rules="rules"
+    :value="value"
+    validate-on-blur
```

Comment:
validates when user moves away (on blur)

Reply:
Still doesn't work as expected: I see a validation error as soon as I enter "2" (for 2022). I should see the error after I leave the component (ie, lazy-validation if this was in a v-form), or when I click File Now.

Reply:
Please try again. code should be updated now

Reply:
Yes, works perfectly now 👍